snippets/vectorAngle.md

```diff
@@ -0,0 +1,36 @@
+---
+title: vectorAngle
+tags: math,intermediate
+---
+
+Returns the angle (theta) value between two vectors.
+
+Use `Math.trunc()` to pick the integer value of length. Use `slice()` slice the incoming array into individual vectors. 
+Use `forEach()`, `Math.pow()` and `Math.sqrt()` to iterate over the vectors and calculate the magnitude of each vector and the scalar product.
+Use `Math.acos()` to calculate arccos and get the theta.
+
+
+```js
+const theta = ((...x) =>{ 
```

Comment:
There's a wrapping parenthesis around the whole function that is unnecessary.

Reply:
Done